uPortal-portlets/build.gradle

```diff
@@ -6,6 +6,11 @@ dependencies {
     compile project(':uPortal-web')
 
     compile "com.amazonaws:aws-java-sdk-s3:${awsVersion}"
+//    constraints {
+//        implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:${jacksonVersion}") {
+//            because 'previous versions have security problems'
+//        }
+//    }
```

Comment:
do we still need this commented out code?
could it be removed?

Reply:
The comment provide an example and a description of what to do, so in my mind it's required to keep it, as only some peoples deploy this lib (nobody in France at least).

Reply:
The AWS SDK lib seems to be needed even if we don't use it to be able to compile, but it provides a transient lib that isn't required but I tested removing it without having a context using AWS SDK. So I don't know if it works in AWS SDK context when removing this transient dependency.
After removing this transient dependency is a "security" patch (CVE at low-level) when we don't need it.
Also keeping this comment could provide a warning point to someone who know how to test that.

After the question is how to manage de potential security problem at this point ?

Reply:
Looking though the known vulns https://security.snyk.io/vuln/maven/?search=Jackson
None apply to the current version of Jackson used by AWS SDK https://github.com/aws/aws-sdk-java/blob/04eaedd554094f734967ecee8b54cd938ccbd55f/pom.xml#L362

And we are running latest AWS SDK which has these patches https://github.com/uPortal-Project/uPortal/pull/2542

Unless you're aware of an unreported vulnerability, I don't think there is a potential vuln here.
Which means a warning seems unwarranted.